src/App.tsx

```diff
@@ -86,8 +88,28 @@ const App = (): JSX.Element => {
                     setIsOpen={setDevtoolsOpen}
                     onDragStart={(_) => {}}
                   />
-
-                  <div className="bg-blue-primary"></div>
+                  <div className="bg-blue-primary pl-1 text-white-primary">
+                    <div>
+                      <input
+                        type="checkbox"
+                        id="showIDs"
+                        onChange={(e) => setShowIDs(e.target.checked)}
+                        className="mr-1"
+                      />
+                      <label htmlFor="showIDs">show node IDs</label>
```

Comment:
You mention in your commit message that 
> Note that, for now at least, we only show these for the "primer", "primer-simple" and "primer-box" nodes. For others, it's less obvious where to put them, since we don't have labels as such.

Some off-the-cuff ideas on where to put them if we want:
- mouse over/hover text
- some sort of popup on click
- click toggles display of normal node or just its id
- in the sidebar or info display when node is selected

Having said that, I don't currently see a pressing need for them. Maybe leave it as-is until there is a situation where we want to see them?

FTR, one (painful!) way to access these IDs is to inspect the node in your browser's devtools and then find the `data-id` attribute on the containing `react-flow__node` node. (At least, I think this works, I haven't tested it much)